site/en/codelabs/silabs-openthread-hardware/index.lab.md

```diff
@@ -0,0 +1,461 @@
+
+3. The Project Configuration dialog opens. Here you can rename your project, change the default project file location, and determine if you will link to or copy project files. Linked project files point to the SDK and any modifications you make end up being made in the SDK and being used for future projects. Copying project sources, allows you to edit a project-local copy so SDK files remain intact. '**Link sdk and copy project sources**' is the default and the recommended choice. Click **FINISH**.
+
+![New project wizard step 3](./img/new_project_wizard_3.jpg)
+
+4. The Simplicity IDE Perspective opens with the Project Configurator open to the **OVERVIEW** tab.
+
+![Project overview](./img/new_project_wizard_4.png)
+
+The project is configured from the **Software Components** tab by installing and uninstalling components, and configuring installed components. Installed components are checked. Click **Installed Components** to see a filtered list of components installed by the example application. Any changes you make are autosaved, and project files are autogenerated. Progress is shown in the lower right corner of the Simplicity IDE perspective.
+
+![Software Components](./img/software_components.png)
+
+For this demo we are going to use the default configuration of the sample applications. Repeat the above steps to create the `ot-rcp` project for your other board.
+
+### Build and Flash the projects
+
+Build and flash both `ot-rcp` and `ot-cli-ftd` projects.
+
+1. After your project is configured, click the **Build control (hammer icon)** in the top tool bar. Alternatively you can right-click on the project and click build project.
+
+![Build project button](./img/build_button.png)
+
+2. Progress is displayed in the **Console** and a progress bar in the lower right. Any errors or warnings related to your project will also appear in this output window.
+
+![Porject build output window](./img/project_build_output.png)
+
+3. The binaries images are generated after a successful project build. You can flash the binary image from the **Project Explorer view**. Locate the .bin, .hex, or .s37 file in the compiler subdirectory. **Right-click the file** and select **Flash to Device**. If you have more than one devices connected then **select a device** to program, click **OK**. The Flash Programmer opens with the file path populated. Click **PROGRAM**.
+
+![Flash](./img/flash_device_selection.png)
+
+## Firmware set up summary
+
+Duration: 1:00
+
+At this point you should have created, compiled and flashed the appropriate firmware on the radio boards. Once `ot-rcp` is flashed to BRD4166A, disconnect it from the host system and connect this board to the Raspberry Pi.
+
+After completing this section, your Thread network hardware set up will look like this.
+
+> Not shown in the image is the WSTKs connected to a Windows/Linux/Mac Host system via USB and the Raspberry Pi 3B+ is connected to the internet using an Ethernet cable. 
```

Comment:
Add this before the note

<noparse>> aside positive</noparse>
<noparse>></noparse>